### DISCUSSION TITLE
TRACKER_SCRIPT_NAME not works on netlify deployment

### DISCUSSION THREAD

**Discussion by vzchn** [umami-software/umami#1336] [General]
I set TRACKER_SCRIPT_NAME  in delply env, and re-deploy twice, still not works:
1. the tracker url does not changes in unami admin panel.
2. It shows a 404 page on admin panel when try to use modified tracker url(set in the env).

**Comment by mikecao** [comment]
The TRACKER_SCRIPT_NAME depends on Next.js middleware to work. Not sure if Netlify supports that. According to this post they should now, https://www.netlify.com/blog/deploy-nextjs-12-2/
